requirements.txt

```
@@ -32,9 +30,4 @@ wrapanapi==3.6.0
 # Get airgun, nailgun and upgrade from master
 git+https://github.com/SatelliteQE/airgun.git@master#egg=airgun
 git+https://github.com/SatelliteQE/nailgun.git@master#egg=nailgun
```

newer style dependency definition
```suggestion
airgun @ git+https://github.com/SatelliteQE/airgun.git@master#egg=airgun
nailgun @ git+https://github.com/SatelliteQE/nailgun.git@master#egg=nailgun
```